packages/webpack/src/HoneybadgerSourceMapPlugin.js

```diff
@@ -0,0 +1,133 @@
+import async from 'async';
+import request from 'request';
+import VError from 'verror';
+import find from 'lodash.find';
+import reduce from 'lodash.reduce';
+import { handleError, validateOptions } from './helpers';
+import { ENDPOINT, PLUGIN_NAME } from './constants';
+
+export default class HoneybadgerSourceMapPlugin {
```

Comment:
I added `export default` here and now we have `module.exports` and `export default` in this file.
The reason is that our tests use `import`, but in docs we have instructions with `require()` and eslint complained that a default export is not found.
I tested both import and require and they work, so I left them both to ensure backwards compatibility. What do you think?

Reply:
The `honeybadger-webpack` repository uses [standard](https://github.com/standard/standard) and not `eslint` (even though there is an `.eslintignore` file 🤷).

Btw, even with eslint complaining, the tests pass and that confuses me 😅 

Here's how the compiled output looks:
```
exports.default = HoneybadgerSourceMapPlugin;
module.exports = HoneybadgerSourceMapPlugin;
```

Previously, it was only:
```
module.exports = HoneybadgerSourceMapPlugin;
```

Reply:
I see. Eh, I guess as can take it and worry about making it "proper" later.

Reply:
But on the flip side, that's a strange rule and we should probably look into it. I know Webpack only uses CommonJS, so `module.exports` is enough for any Webpack plugin. I think the integration between TypeScript and ESLint is off here (maybe that rule needs to be disabled for this file?)

Reply:
OK, I think I know why this is not working: Our eslint is configured globablly for all packages in the monorepo (for better consistency). It is configured with a Typescript parser, but this package does not have a tsconfig, nor typescript files. I'm pretty sure this is related to this problem.
For the moment, I have decided to just ignore the error like you suggested and will fix it by introducing Typescript in the package.